### ISSUE TITLE
Product sales graph does not match orders count/items sold

### ISSUE THREAD

**Issue by stephhjacq** [woocommerce/woocommerce-admin#2719]
**Describe the bug**
Product sales graph does not match orders count/items sold.

**To Reproduce**

For this case, the storeholder needs the flexibility to change a simple product to a variable product while tracking historic sales.

1. Look at report for simple product, SKU 987, sales month to date under Analytics > Product
2. Edit product, making it variable. Use SKU 987 for one of the variations
3. Purchase variation 987
4. View report for product by title, sales month to date. Totals are correct, but graph no longer shows the historic purchase

**Expected behavior**
Graph should match data.

**Screenshots**
Simple product: Sticker (SKU 987), before

![https://cld.wthms.co/q5xBiC](https://cld.wthms.co/q5xBiC+)
Full Image: https://cld.wthms.co/q5xBiC+

Variable product: Sticker - Laptop variation (SKU 987), after

![https://cld.wthms.co/gwqfYA](https://cld.wthms.co/gwqfYA+)
Full Image: https://cld.wthms.co/gwqfYA+

**Desktop (please complete the following information):**

* OS: Mac OS 10.14.5
* Browser Chrome Version 75.0.3770.142 (Official Build) (64-bit)

**Comment by psealock** [comment]
Thanks for the bug report @stephhjacq.

I'd to clarify one thing to make sure I understand correctly. In the second screenshot `Variable product: Sticker - Laptop variation (SKU 987), after`, you'd like to see the previous order as well. In other words, "Items Sold" is correct with a number of 2, but the chart and table don't show the original purchase. Is that right?

This is an interesting question on how to handle changed products and historical data. At the very least, there should be cohesion between the summary numbers and chart/table.

**Comment by nagpai** [comment]
>I'd to clarify one thing to make sure I understand correctly. In the second screenshot Variable product: Sticker - Laptop variation (SKU 987), after, you'd like to see the previous order as well. In other words, "Items Sold" is correct with a number of 2, but the chart and table don't show the original purchase. Is that right?

I think what the user needs is the orders placed for the `Simple product - Sticker`  before it got converted to `Variable product - Sticker - Laptop variation`

It seems that the system treats the variable product independent of the legacy simple product. One work around for now is to clone the simple product (change it to draft) and make variants from the cloned product.

**Comment by JScott-MSI** [comment]
I am the original reporter of this issue with support. I have some additional information that might clarify the issue that I am seeing.

I converted a product from simple to variable to see if the reporting would carry forward. If I look at the converted product using the "classic" Report the product seems to show the correct sales information. (see image) you'll also see that the variant has been assigned an id number.

![Classic report](https://user-images.githubusercontent.com/52297130/62189368-9d8cfa00-b323-11e9-8bd9-7a3da2277b98.jpeg)

When I try to look at the product history using the new Admin analytics tools it shows me the summary numbers but doesn't show the graph or product details properly. (see image)

![post conversion](https://user-images.githubusercontent.com/52297130/62189552-1be99c00-b324-11e9-8709-ba3596e50078.jpeg)

It also shows correctly in the All Products list after conversion. I blanked out the revenue totals on purpose, the one that is still showing is the converted product.

![Shows correctly](https://user-images.githubusercontent.com/52297130/62189759-b0ec9500-b324-11e9-93fc-2c7cc2f8f2a5.jpeg)

I'm not sure if this is an issue with creating the graph and table or if it's the suggestions stated above.
